tests/func-tests/imx/step-definitions/burning.ts

```diff
@@ -1,6 +1,12 @@
 import { UnsignedTransferRequest } from '@imtbl/core-sdk';
-import { ImmutableXClient, ImxClientModuleConfiguration } from '@imtbl/sdk/immutablex_client';
-import { GenericIMXProvider, ProviderConfiguration } from '@imtbl/sdk/provider';
+import {
+  ImmutableXClient,
+  ImxClientModuleConfiguration,
+} from '@imtbl/sdk/x_client';
```

Comment:
what makes one not import from `@imtbl/x-client` but has a `sdk` in the path?

Reply:
The functional tests use the SDK like our external customers.